pymoveit2/robots/kinova.py

```diff
@@ -0,0 +1,32 @@
+from typing import List
+
+MOVE_GROUP_ARM: str = "arm"
+MOVE_GROUP_GRIPPER: str = "gripper"
+
+JACO_PREFIX: str = "j2"
+MICO_PREFIX: str = "m1"
+
+def get_prefix(version_prefix: str = JACO_PREFIX, arm_dof: int = 6, hand_dof: int = 2) -> str:
+    return version_prefix + "n" + str(arm_dof) + "s" + str(hand_dof) + "00_"
```

Comment:
I believe the 3rd character can either be `n` or `s` based on the wrist type (spherical or non-spherical), where I believe the `s` models are more common. Same with the robot mode (5th character), which also seems that it can vary (`s`/`a`) - although I have only seen `s`.

See further [explanation (docs)](https://github.com/Kinovarobotics/kinova-ros/blob/924781d3dfe241b2b94b3c72a804b80d3658cf02/README.md#launch-driver).

Reply:
Good catch, thank you! Updated.

Reply:
Thanks!